samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,351 @@
+
+.. _CONFIG_CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS:
+
+CONFIG_CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS - Cloud connection retry timeout (seconds)
+   If connecting to `nRF Cloud`_ takes longer than this timeout, it will be reattempted.
+
+.. _CONFIG_CLOUD_CONNECTION_REESTABLISH_DELAY_SECONDS:
+
+CONFIG_CLOUD_CONNECTION_REESTABLISH_DELAY_SECONDS - Cloud connection reestablishment delay (seconds)
+   After the connection to `nRF Cloud`_ has been reset, wait this amount of time before attempting to connect again.
+
+.. _CONFIG_CLOUD_READY_TIMEOUT_SECONDS:
+
+CONFIG_CLOUD_READY_TIMEOUT_SECONDS - Cloud readiness timeout (seconds)
+   If the connection to `nRF Cloud`_ does not become ready within this timeout, the sample will reset its connection and try again.
+
+.. _CONFIG_DATE_TIME_ESTABLISHMENT_TIMEOUT_SECONDS:
+
+CONFIG_DATE_TIME_ESTABLISHMENT_TIMEOUT_SECONDS - Modem date and time establishment timeout (seconds)
+   The sample will wait this many seconds for the modem to determine the current date and time before giving up and moving on.
+
+.. _CONFIG_APPLICATION_THREAD_STACK_SIZE:
+
+CONFIG_APPLICATION_THREAD_STACK_SIZE - Application Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the application thread of the sample.
+
+.. _CONFIG_CONNECTION_THREAD_STACK_SIZE:
+
+CONFIG_CONNECTION_THREAD_STACK_SIZE - Connection Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the connection thread of the sample.
+
+.. _CONFIG_MESSAGE_THREAD_STACK_SIZE:
+
+CONFIG_MESSAGE_THREAD_STACK_SIZE - Message Queue Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the message queue processing thread of the sample.
+
+.. _CONFIG_MAX_OUTGOING_MESSAGES:
+
+CONFIG_MAX_OUTGOING_MESSAGES - Outgoing message maximum
+   Sets the maximum number of messages that can be in the outgoing message queue.
+   Messages submitted past this limit will not be enqueued.
+
+.. _CONFIG_MAX_CONSECUTIVE_SEND_FAILURES:
+
+CONFIG_MAX_CONSECUTIVE_SEND_FAILURES - Max outgoing consecutive send failures
+   Sets the maximum number of device messages which may fail to send before a connection reset and cooldown is triggered.
+
+.. _CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS:
+
+CONFIG_CONSECUTIVE_SEND_FAILURE_COOLDOWN_SECONDS - Cooldown after max consecutive send failures exceeded (seconds)
+   If a connection reset is triggered by too many failed device messages, the sample will wait this long (in seconds) before trying again.
+
+.. _CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS:
+
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS - Sensor sampling interval (seconds)
+   Sets the time to wait between each temperature sensor sample.
+
+.. warning::
+   Every time a sensor sample is sent, it will interrupt any attempted GNSS fix.
+   If you set this interval too short, there will not be enough time to ever acquire a fix.
+   In good conditions, and with AGPS data, one minute is generally just barely long enough to get a fix most of the time.
```

Comment:
```suggestion
.. note::
   Every time a sensor sample is sent, it interrupts any attempted GNSS fix.
   If you set this interval too short, there is not enough time to acquire a fix.
   In good conditions, and with A-GPS data, one minute is generally just barely long enough to get a fix most of the time.
```
We are not supposed to use warnings, unless a task or process is life threatening.

Reply:
Oh, my bad. Good to know, thank you.